contracts/shared/Types.sol

```diff
@@ -17,3 +17,12 @@ struct AftermarketDeviceOwnerPair {
     uint256 aftermarketDeviceNodeId;
     address owner;
 }
+
+struct MintVirtualDeviceInput {
+    uint256 integrationNode;
+    uint256 vehicleNode;
+    bytes virtualDeviceSig;
```

Comment:
I don't think we're going to have the virtual device sign. @zer0stars can you confirm this?

Reply:
Why the address then if not for this?